.github/workflows/node.js.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ${{matrix.os}}
     strategy:
       matrix:
-        node-version: [16.x, 18.x]
+        node-version: [16.x, 18.x, 20.x]
```

Comment:
Maybe should we get rid of 16?

https://github.com/nodejs/release#release-schedule

September is coming :)

Reply:
Yeah, nice catch ;)